book.rb

```diff
@@ -0,0 +1,16 @@
+class Book
+  attr_reader :rentals
+
+  def initialize(author:, title:)
+    @author = author
+    @title = title
+    @rentals = []
+  end
+
+  attr_accessor :author, :tittle
+
+  def add_rental(rental)
+    @rentals << rental
+    rental.book = self
+  end
```

Comment:
- [ ] As requested by the previous code reviewer, which is a requirement of this project, The `add_rental` method takes `person`  and `date` as  parameters, and `book` itself is sent as a parameter to create the new rental achieving the 'has-many' association. Something like so: 

```ruby
  def add_rental(person, date)
    Rental.new(date, self, person)
  end
```

Please kindly fix that.